splice/commands.py

```diff
@@ -265,6 +266,59 @@ def ingest_tiles(in_file, channel_id, out_path, console_out, deploy_flag, verbos
         traceback.print_exc()
 
 
+@DataCommand.option("-v", "--verbose", action="store_true", dest="verbose", help="turns on verbose mode", default=False, required=False)
+@DataCommand.option("-q", "--quiet", action="store_true", dest="quiet", help="turns on quiet mode", default=False, required=False)
+@DataCommand.option("-l", "--leniency-minutes", type=int, dest="leniency", help="Leniency period in minutes for scheduling", default=15)
```

Comment:
The reason why you need it is because clocks are not perfect, especially when running a scheduled job as a cron job.

The projected use case is to run a cron job every 15 minutes. Every scheduled distribution that is set within the past 15 minutes, up to the present minute, are candidates to be deployed.

If there were no leniency allowed, the command would only schedule jobs that have the exact same datetime as the time at which the command was run. Even a deviation of a second would prevent scheduled jobs from running.

There are more complicated methods to do this, e.g. rounding down the execution time to the nearest hour chunk. I felt that a leniency period was a simple way around this.

Either way, the cron job is set to run from only one host.